packages/flame_riverpod/lib/src/widget.dart

```diff
@@ -47,14 +51,31 @@ class RiverpodAwareGameWidgetState<T extends Game> extends GameWidgetState<T>
   List<_ListenManual<Object?>>? _manualListeners;
 
   /// Rebuilds the [RiverpodAwareGameWidget] by calling [setState].
+  // Undesirable to call [setState] while the widget may be building.
+  // Honour requests to rebuild by setting a flag.
```

Comment:
I guess this is meant as a comment and not a dartdoc? Never seen them mixed after each other before, does it even work? :sweat_smile:

Reply:
Maybe the comments can be moved into the function?

Reply:
I had considered that but have just modified the comments outside the function!